### ISSUE TITLE
Keyboard Focus and Accessibility Focus should be the same

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#9826]
### Problem Description

According to UIA and Narrator team, on Windows, keyboard focus and accessibility focus should always point to the same element. Currently in RNW, when a focusable element is passed `accessible=false`, the element is removed from the UIA tree but remains keyboard focusable. Thus the user, when tabbing to an element with `accessible=false` with a screen reader enabled, can see keyboard focus move to the element and accessibility focus jump to the app window. Then, the screen reader announces `<app name> window` creating a poor/disorienting screen reader experience. 

### Steps To Reproduce

Run the Playground App.
Edit a sample page to include a TouchableHighlight control and with the `accessible` set to false. 
Enable narrator.
Tab to TouchableHighlight.

### Expected Results

When an element is passed `accessible=false`, it should no longer be `focusable` leading both keyboard focus and the screen reader to skip over the element. 

### CLI version

N/A

### Environment

```markdown
N/A
```

### Target Platform Version

10.0.19041

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2019

### Build Configuration

Debug

**Comment by AgneLukoseviciute** [comment]
related: https://github.com/microsoft/react-native-windows/issues/8875
